graphbolt/CMakeLists.txt

```diff
@@ -85,6 +87,12 @@ if(BUILD_WITH_TASKFLOW)
   target_compile_definitions(${LIB_GRAPHBOLT_NAME} PRIVATE BUILD_WITH_TASKFLOW=1)
 endif()
 
+if(USE_OPENMP)
```

Comment:
@frozenbugs regression was caused by not enabling OpenMP for GraphBolt. This is the only change in this PR compared to the previous one. In addition, I am removing unused options from GraphBolt's build.sh.